kqueen/engines/openstack_kubespray.py

```diff
@@ -822,7 +824,7 @@ def shrink(self, *, resources, remove_hostnames):
         return resources
 
     def _get_userdata(self):
-        docker_bip = {"bip": self.meta["docker_bip_network"]}
+        docker_bip = {"bip": self.cluster.metadata.get("docker_bip_network", "10.13.0.1/16")}
```

Comment:
if u want to define "10.13.0.1/16" as sane default, please define it in kqueen config base.py